doc/data/messages/e/empty-docstring/good.py

```diff
@@ -0,0 +1,2 @@
+def foo():
+    """A dummy description."""
```

Comment:
```suggestion
def add(a: Any, b: Any) -> Any:
    """Add a and b."""
    return a + b
```

Reply:
I mean, what does adding `Any` really provide here? Without any information `mypy` and others will already infer everything as `Any`.